src/main/java/teamroots/embers/tileentity/TileEntitySteamEngine.java

```diff
@@ -39,7 +42,7 @@
 import java.util.List;
 
 public class TileEntitySteamEngine extends TileEntity implements ITileEntityBase, ITickable, ISoundController, IExtraCapabilityInformation {
-    class BurningFuel {
+    static class BurningFuel {
```

Comment:
why static?

Reply:
IntelliJ suggests it, also I didn't touched it in my config refactor.